src/tools/genpolicy/src/registry.rs

```diff
@@ -153,10 +154,22 @@ impl Container {
                 .await
                 .unwrap();
 
+                // Find the last layer with an /etc/passwd file,
+                // respecting whiteouts.
+                let mut passwd = "".to_string();
```

Comment:
We need to later query passwd to update GIDs, so more convenient to store it in the Container.